<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DlgAudioSet</class>
 <widget class="QDialog" name="DlgAudioSet">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>373</width>
    <height>338</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>音频设置</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#DlgAudioSet {
	background: rgb(255,255,255);
}
QComboBox {
	background: transparent;
	border: 1px solid rgb(221,221,221);
	border-radius: 4px;
	min-height: 30px;
	padding-left: 8px;
	padding-right: 8px;
}
QComboBox:hover {
	border-color:rgb(57,129,252);
}
QComboBox::drop-down {
	padding-top: 5px;
	image: url(:/Resources/dropDown.png);
}
QComboBox QAbstractItemView{
	margin-top:1;
	outline:0;	
	border:1px solid rgb(225,225,225);
	background-color:rgb(255,255,255);
	selection-background-color:rgb(80,145,255);
}
QComboBox QAbstractItemView::item{
	min-height:30px;
	max-height:30px;
}
QComboBox QAbstractItemView::item:hover{
    background-color:rgb(240,240,240);
	color:rgb(45,45,45);
}
QComboBox QAbstractItemView::item:selected{
    background-color:rgb(240,240,240);
	color:rgb(45,45,45);
}
QSlider {
	min-height: 20px;
	max-height: 20px;
}
QSlider::add-page:horizontal {       
	background-color:rgb(223,223,223);  
}  
QSlider::sub-page:horizontal {  
	background-color: rgb(57,129,252);
}
QSlider::sub-page:horizontal:disabled {  
	background-color: rgb(223,223,223);
}  
QSlider::groove:horizontal {  
	background:transparent;  
	height:2px;
}  
QSlider::handle:horizontal {
	height: 12px;
	width: 12px;
	margin-top: -5px;
	margin-bottom: -5px;
	border-radius: 6px;
	background: rgb(57,129,252);
}  
QSlider::handle:horizontal:disabled {
	background: rgb(223,223,223);
}  </string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>10</number>
   </property>
   <property name="topMargin">
    <number>10</number>
   </property>
   <property name="rightMargin">
    <number>10</number>
   </property>
   <property name="bottomMargin">
    <number>10</number>
   </property>
   <property name="verticalSpacing">
    <number>8</number>
   </property>
   <item row="5" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>选择扬声器：</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>选择麦克风：</string>
     </property>
    </widget>
   </item>
   <item row="6" column="1">
    <widget class="QLabel" name="label_5">
     <property name="text">
      <string>音量：</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1" colspan="2">
    <widget class="QComboBox" name="cbBx_micSel"/>
   </item>
   <item row="1" column="2">
    <widget class="QSlider" name="sld_micVol">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="6" column="2">
    <widget class="QSlider" name="sld_spkVol">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>音量：</string>
     </property>
    </widget>
   </item>
   <item row="5" column="1" colspan="2">
    <widget class="QComboBox" name="cbBx_spkSel"/>
   </item>
   <item row="2" column="1" colspan="2">
    <widget class="QCheckBox" name="agc">
     <property name="text">
      <string>自动调节麦克风音量</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1" colspan="2">
    <widget class="QCheckBox" name="ans">
     <property name="text">
      <string>自动降噪</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1" colspan="2">
    <widget class="QCheckBox" name="aec">
     <property name="text">
      <string>自动回声消除</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>cbBx_micSel</tabstop>
  <tabstop>sld_micVol</tabstop>
  <tabstop>agc</tabstop>
  <tabstop>ans</tabstop>
  <tabstop>aec</tabstop>
  <tabstop>cbBx_spkSel</tabstop>
  <tabstop>sld_spkVol</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
